### DISCUSSION TITLE
Form tab 1 in form tab 2

### DISCUSSION THREAD

**Discussion by magiaslab** [Budibase/budibase#7757] [General]
Hello everyone, I am building a company system for the registration of transport documents of material entering or leaving a warehouse. Each packing slip contains some unique data and a row for each incoming item. I created the tables ddtin ddtout the database of the archives and a table of movements where I would like to link the list of items to the document by association.
The question is how can I create the article entry form (Table 2) on the document screen (Table 1)?

**Comment by melohagan** [comment]
Hey @magiaslab 

Do any of these pages help?

 - https://docs.budibase.com/docs/fetching-data#displaying-relational-data
 - https://docs.budibase.com/docs/url-parameters
 - https://docs.budibase.com/docs/form-steps
 - https://docs.budibase.com/docs/referencing#example-creating-and-linking-two-table-entries-in-one-form

**Comment by magiaslab** [comment]
Thanks @melohagan  what I'm trying to do is slightly different but the tutorials are still useful.

My transport document that I need to register contains:

- The customer data that I have saved in a CUSTOMER table
- Some single data such as date, document number etc.
- The items that I have to load into the warehouse for which I have created a master data table and created a relationship with this table.

When a document arrives I register it and connect the customer and the articles and so far everything is ok through the relationships. In each document, however, I must enter data relating to the articles, such as quantity, price, etc. I would like to do it in the same form with a multiline entry but I don't know how to create it.

I thought of creating a third "movements" table where I link the items to the posting of the transport document by ID.

Thanks for your help
